src/lib/devConsole.js

```diff
@@ -886,6 +886,64 @@ const getTenantFeatures = async organizationCode => {
 	}
 };
 
+/**
+ * Gets the deployments value for mesh.
+ *
+ * This request bypasses the Dev Console and is sent directly to the Schema Management Service.
+ * As a result, we provide the orgCode instead of orgId since Dev Console usually performs the translation.
+ * The near-term goal is to stop using Dev Console as a proxy for all routes.
+ * @param organizationCode
+ * @param projectId
+ * @param workspaceId
+ * @param meshId
+ * @returns {Promise<Object>}
+ */
+const getMeshDeployments = async (organizationCode, projectId, workspaceId, meshId) => {
+	const { accessToken, apiKey } = await getDevConsoleConfig();
+	const config = {
+		method: 'get',
+		url: `${SMS_BASE_URL}/organizations/${organizationCode}/projects/${projectId}/workspaces/${workspaceId}/meshes/${meshId}/deployments/latest?API_KEY=${apiKey}`,
```

Comment:
Quick question @nvyasadobe, So here do we eventually plan to route through DEV_CONSOLE_BASE_URL and you have SMS_BASE_URL for testing for the time being?

Reply:
No, these will always go directly to SMS.
The new tenants route and deployments route bypass devconsole. We are moving away from devconsole